src/recordlinker/linking/matchers.py

```diff
@@ -177,15 +148,14 @@ def compare_probabilistic_fuzzy_match(
         # Return early if a field is missing, and log that was the case
         return (missing_field_points_proportion * log_odds, True)
 
-    similarity_measure, threshold = _get_fuzzy_params(str(key.attribute), **kwargs)
```

Comment:
Just adding on to the above comment, the uniformity is why we had this `get_params` functions which didn't do a whole lot else. We don't necessarily need to use this, though, just adding some context.

Reply:
(now irrelevant, gonna resolve this because the first implementation was cleaner)